.github/workflows/playwright.yml

```diff
@@ -30,12 +30,14 @@ jobs:
         with:
           php-version: "8.1"
           tools: composer
-      - uses: subosito/flutter-action@v2.5.0
+      - uses: subosito/flutter-action@v2.8.0
         with:
-          flutter-version: "2.8.1"
+          flutter-version: "3.3.4"
           channel: "stable"
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
       - run: npm install
-      - run: dart pub global activate dhttpd
```

Comment:
I have removed the dart `dhttpd` package and replaced it with a simple python3 web server. It seems this was causing a lot of issues for the CI to flake on the tests. 

I also changed out the flutter tests from using the proxy to using the ory tunnel and bumped the ory cli version to the latest.